src/webots/core/WbTemplateEngine.cpp

```diff
@@ -68,7 +68,7 @@ void WbTemplateEngine::copyModuleToTemporaryFile(QString modulePath) {
 
 void WbTemplateEngine::initializeJavaScript() {
   // copy JavaScript modules to the temporary directory
-  QDirIterator it(WbStandardPaths::resourcesPath() + "javascript/", QDirIterator::Subdirectories);
+  QDirIterator it(WbStandardPaths::resourcesPath() + "web/wwi/protoVisualizer/templating/", QDirIterator::Subdirectories);
```

Comment:
just a doubt, the folder protoVisualizer is included in the distribution right? Otherwise this won't work. Basically we moved the js classes used in procedural proto to a new location in order to avoid having 2 copies of it. (needed by webots and proto viewer to evaluate template statements)

Reply:
Yes we included it.
https://github.com/cyberbotics/webots/blob/0c16a72c33534d41f14a65a9fa31bf73e9015f1c/scripts/packaging/files_core.txt